src/Utils/recaptcha.ts

```diff
@@ -1,30 +1,40 @@
-import { data as sd } from "sharify"
+import { getENV } from "Utils/getENV"
 import createLogger from "Utils/logger"
 
 const logger = createLogger("recaptcha.ts")
 
-export const recaptcha = (action: RecaptchaAction, cb?: any) => {
-  if (sd.RECAPTCHA_KEY) {
+// TODO: Should return a Promise instead of accepting a callback
+export const recaptcha = (
+  action: RecaptchaAction,
+  callback?: RecaptchaCallback
+) => {
+  if (getENV("RECAPTCHA_KEY")) {
```

Comment:
👍 (though we might want to update citadel with this so folks always have it)

Reply:
I didn't add this, just changed it to use `getENV`. Should be there already.